vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ReferenceManager.java

```diff
@@ -42,6 +52,13 @@ public IReference createReference(String biblioId, String title, String author,S
         reference.setEditors(editor);
         reference.setType(type);
         reference.setNote(note);
+        if(visibility == "Private") {
```

Comment:
strings should not be hard-coded; at a minimum this needs to be a constant somewhere